src/deck/drivers/src/Kconfig

```diff
@@ -93,6 +93,22 @@ config DECK_LEDRING
       control from your computer. Two additional strong front facing
       LEDs that can be switched on and off, act as headlights.
 
+config DECK_LEDRING_DEFAULT_EFFECT
+    int "Default light effect to use on the LED ring"
+    depends on DECK_LEDRING
+    default 6
+    help
+      Index of the default light effect that the LED ring should use
+      after boot. Use zero to turn off the LEDs when booting.
```

Comment:
Would it make sense to list what the effects are here? Or is that superfluous?

Reply:
I wouldn't list them here, maybe point the user to the appropriate header file instead. If the list of effects change, this is the first place that everyone will forget to update :)

Reply:
Agreed, perhaps do something similar as in this help text: https://github.com/bitcraze/crazyflie-firmware/blob/master/Kconfig#L34

Reply:
Can you help me with finding the page to link to? Or is it okay to just link to bitcraze.io in general?

Reply:
> Can you help me with finding the page to link to? Or is it okay to just link to bitcraze.io in general?

I think just to https://www.bitcraze.io/ is what we said, right @knmcguire, since pages can move around.

"[...] for more information about the LED ring effects see the parameter documentation on https://www.bitcraze.io/"

Maybe?

Reply:
Done.